{"uuid":"8edf35be-7c1f-47e0-8931-09aeef7a6b09","historyId":"ea8d783e50d710054672db63ead1a203","fullName":"com.bank.los.tests.LoginTest.validLoginTest","labels":[{"name":"package","value":"com.bank.los.tests.LoginTest"},{"name":"testClass","value":"com.bank.los.tests.LoginTest"},{"name":"testMethod","value":"validLoginTest"},{"name":"parentSuite","value":"Surefire suite"},{"name":"suite","value":"Surefire test"},{"name":"subSuite","value":"com.bank.los.tests.LoginTest"},{"name":"host","value":"FTL123456"},{"name":"thread","value":"49140@FTL123456.main(3)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"story","value":"Login Functionality"},{"name":"severity","value":"critical"}],"links":[],"name":"validLoginTest","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"java.lang.ExceptionInInitializerError","trace":"java.lang.ExceptionInInitializerError\n\tat com.bank.los.core.ui.DriverFactory.initDriver(DriverFactory.java:16)\n\tat com.bank.los.tests.LoginTest.setup(LoginTest.java:27)\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:565)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)\n\tat org.testng.internal.invokers.TestInvoker.runConfigMethods(TestInvoker.java:833)\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:600)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1604)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:739)\n\tat org.testng.TestRunner.run(TestRunner.java:614)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\n\tat org.testng.TestNG.run(TestNG.java:1079)\n\tat org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:155)\n\tat org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:169)\n\tat org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:88)\n\tat org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:137)\n\tat org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)\n\tat org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)\n\tat org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)\n\tat org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)\nCaused by: java.lang.RuntimeException: ‚ùå Failed to load configuration file for environment: qa\n\tat com.bank.los.core.config.ConfigReader.loadConfig(ConfigReader.java:23)\n\tat com.bank.los.core.config.ConfigReader.<clinit>(ConfigReader.java:11)\n\t... 37 more\nCaused by: java.io.FileNotFoundException: core/src/main/resources/config.qa.properties (No such file or directory)\n\tat java.base/java.io.FileInputStream.open0(Native Method)\n\tat java.base/java.io.FileInputStream.open(FileInputStream.java:185)\n\tat java.base/java.io.FileInputStream.<init>(FileInputStream.java:139)\n\tat java.base/java.io.FileInputStream.<init>(FileInputStream.java:109)\n\tat com.bank.los.core.config.ConfigReader.loadConfig(ConfigReader.java:19)\n\t... 38 more\n"},"stage":"finished","description":"Verify that valid user can login successfully","steps":[],"attachments":[],"parameters":[],"start":1760981016084,"stop":1760981016085}